Jenkinsfile

```diff
@@ -146,6 +159,20 @@ podTemplate(cloud: 'openshift', label: 'coreos-assembler', yaml: pod, defaultCon
             return
         } else {
             currentBuild.description = "[${params.STREAM}] ⚡ ${newBuildID}"
+
+            // and insert the parent info into meta.json so we can display it in
+            // the release browser and for sanity checking
+            if (parent_commit && parent_version) {
+                utils.shwrap("""
+                echo '{
+                    "fedora-coreos.parent-version": "${parent_version}",
+                    "fedora-coreos.parent-commit": "${parent_commit}"
```

Comment:
Similarly here, seems way less ugly to do in Groovy.

Reply:
i.e.:

```
def metaPath = "builds/${newBuildID}/${basearch}/meta.json"
meta = readJSON file: metaPath
meta["fedora-coreos.parent-version"] = parent_version
meta["fedora-coreos.parent-commit"] = parent_commit
writeJSON file: metaPath, json: meta
```

or so (not tested)

Reply:
Alternatively you could update it all in `jq` at once like so:

```
jq '. + {"fedora-coreos.parent-commit": "${parent_commit}", "fedora-coreos.parent-version": "${parent_version}"}' < $meta > tmp/meta.json.new
mv tmp/meta.json.new $meta

Reply:
OK yeah, that's definitely much cleaner. Done! :arrow_down: